ClientApp/viking-vault/src/Common/UserIcon.tsx

```diff
@@ -100,7 +100,7 @@ class UserIcon extends React.Component<any, IUserIconState> {
 
   render() {
     return (
-      <div className="dropdown" ref={this.megaMenu}>
+      <div title="Profile" className="dropdown" ref={this.megaMenu}>
```

Comment:
What is megaMenu?

Reply:
The menu with the **_"View Profile"_** and **_"Logout"_** options.

Reply:
It`s confusing, so it should be changed to something meaningful.